README.md

```diff
@@ -1 +1,111 @@
-# other-hash-methods-code-along
+# Higher Level Hash Methods
+
+##Objectives
+* Learn some interesting methods we can call on hashes
```

Comment:
Objectives should be in the SWBAT format. So Students Will Be Able To... So a bit mor specific, which methods are they learning?